nifi-registry-core/nifi-registry-web-ui/src/main/webapp/template.html

```diff
@@ -0,0 +1,34 @@
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one or more
+  ~ contributor license agreements.  See the NOTICE file distributed with
+  ~ this work for additional information regarding copyright ownership.
+  ~ The ASF licenses this file to You under the Apache License, Version 2.0
+  ~ (the "License"); you may not use this file except in compliance with
+  ~ the License.  You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<!DOCTYPE html>
+<html>
+<head>
+    <title>NiFi Registry</title>
+    <base href="/">
+    <meta charset='UTF-8'>
+    <meta name='viewport' content='width=device-width, initial-scale=1'>
+    <meta http-equiv='Content-Type' content='text/html; charset=UTF-8'/>
+
+    <link rel="stylesheet" href="nifi-registry/node_modules/@nifi-fds/core/common/styles/css/flow-design-system.min.css" type="text/css" />
```

Comment:
Can we add this to the vendor.scss?

Reply:
Unfortunately, no. I tried to get that to work. `nifi-fds` references it's fonts via a relative paths with no way to override. This forces consumers to essentially use the css as-is out of the nested node-modules folder.

I commented on an open PR that is converting nifi-fds over to webpack relating to this issue: https://github.com/apache/nifi-fds/pull/14/files/6cb517d6c1eadf732004341b9e219a8c3f028df6#r281332994

Maybe it deserves it's own JIRA?

Reply:
I faced this problem earlier as well. This is the reason why I commented the same SCSS variable usage suggestion earlier here: https://github.com/apache/nifi-fds/pull/13#discussion_r276196089